react/features/invite/actions.js

```diff
@@ -157,6 +164,25 @@ export function invite(
     };
 }
 
+/**
+ * Stores pending DTMF tones to be sent after a phone connection is established. If more than one phone numbers are
+ * invited into the conference then weird things may happen, as there's no way to match participant with the invited
+ * phone number.
```

Comment:
I guess it's hard to avoid for now if multiple phone numbers with dtmf are dialed...

Reply:
Yeah, actually this part where multiple phone numbers are dialed makes it impossible to handle correctly, because it sends to everyone. Maybe then I should make sure the code only works for the first phone number and log an error for the other. But then still it's hard to tell which jigasi belongs to which phone number.

Reply:
With the latest changes to jigasi (this is not deployed anywhere) but jigasi user will add in its presence the jwt user/group of the user that dialed out. Maybe we can add there and the number dialed  ... and use that information and in case if it is missing do that only to the first one as you proposed.
https://github.com/jitsi/jigasi/pull/200
https://github.com/jitsi/jitsi-meet/pull/4680

Reply:
thanks! I will look into that.